docs/REST API/rest-api.md

```diff
@@ -1212,18 +1212,18 @@ Note that when making changes to the `integrations` object, all integrations mus
     "macos_settings": {
       "custom_settings": {
         "path": "path/to/profile1.mobileconfig",
-        "labels": ["Label 1", "Label 2"]
+        "labels_exclude_any": ["Label 1", "Label 2"]
       },
       {
         "path": "path/to/profile2.json",
-        "labels": ["Label 3", "Label 4"]
+        "labels_include_all": ["Label 3", "Label 4"]
```

Comment:
Add a second name for existing `labels` key: `labels_include_all`

Both keys have the same behavior: profile is only applied to hosts that have all labels.

Reply:
We should update this documentation. This is invalid JSON. I think custom_settings is meant to be an array here